### PR TITLE
Add more web app tutorials

### PR DESCRIPTION
**Pull request by convoliution**

### CODE REVIEW

File: client/verta/docs/quickstart.rst [hunk 1]
```diff
@@ -31,10 +31,15 @@ The Verta client supports Python 2.7 & 3.5–3.8!
 3. Obtain your Verta Credentials
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (convoliution) [inline client/verta/docs/quickstart.rst:33]: https://docs.verta.ai/en/docs-ml-tutorials/quickstart.html#obtain-your-verta-credentials

File: client/verta/docs/quickstart.rst [hunk 2]
```diff
@@ -31,10 +31,15 @@ The Verta client supports Python 2.7 & 3.5–3.8!
...
 
-* On Verta Core or Verta Enterprise, log into the Verta Web App (e.g., `https://app.verta.ai <https://app.verta.ai>`__) and visit the Profile page to find your developer key.
```
Comment (convoliution) [inline client/verta/docs/quickstart.rst:34]: https://docs.verta.ai/en/docs-ml-tutorials/quickstart.html#obtain-your-verta-credentials

File: client/verta/docs/tutorials/custom_users.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
+Creating custom users
```
Comment (convoliution) [inline client/verta/docs/tutorials/custom_users.rst:1]: https://docs.verta.ai/en/docs-ml-tutorials/tutorials/custom_users.html

File: client/verta/docs/tutorials/organizations/access_through_client.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
+Access through the client
```
Comment (convoliution) [inline client/verta/docs/tutorials/organizations/access_through_client.rst:1]: https://docs.verta.ai/en/docs-ml-tutorials/tutorials/organizations/access_through_client.html

File: client/verta/docs/tutorials/sharing.rst [hunk 5]
```diff
@@ -0,0 +1,27 @@
+Sharing and collaboration
```
Comment (convoliution) [inline client/verta/docs/tutorials/sharing.rst:1]: https://docs.verta.ai/en/docs-ml-tutorials/tutorials/sharing.html

File: client/verta/docs/quickstart.rst [hunk 6]
```diff
@@ -31,10 +31,15 @@ The Verta client supports Python 2.7 & 3.5–3.8!
...
+* On Verta Core or Verta Enterprise, log into the Verta Web App (e.g., https://app.verta.ai) and visit the Profile page to find your developer key.
```
Comment (conradoverta) [inline client/verta/docs/quickstart.rst]:
```suggestion
* On Verta Enterprise, log into the Verta Web App (e.g., https://app.verta.ai) and visit the Profile page to find your developer key.
```

**Review by conradoverta — APPROVED** [review]
This is great!

**Pull request merged**